recipe/meta.yaml

```diff
@@ -47,33 +50,29 @@ outputs:
         - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
       host:
         - gtk3 >=3.24.0
-        - glibmm >=2.54.0
-        - cairomm >=1.12.0
-        - pangomm >=2.38.2
+        - cairomm-1.0
+        - pangomm-1.4
         - sigcpp-2.0
-        - atkmm >=2.24.2
-        - cairo
+        - atkmm-1.6
+        - glibmm-2.4
         - glib
-        - atk
-        - gdk-pixbuf >=2.35.5
-        - pango
-        - glib
-      run:
-        - {{ pin_compatible("pango") }}
-        - {{ pin_compatible("glibmm") }}
       run_constrained:
         - gtkmm {{ version }}
     test:
       commands:
-        - test -f ${PREFIX}/lib/libgtkmm-{{ abi_version }}${SHLIB_EXT}
+        - test -f ${PREFIX}/lib/libgtkmm-{{ abi_version }}${SHLIB_EXT}  # [unix]
+        - if not exist %LIBRARY_BIN%\\gtkmm-vc142-{{ abi_version }}-1.dll exit 1  # [win]
```

Comment:
Can we patch the `vc142` out?

Reply:
Otherwise we'll have to pin to the major.minor version of `vc`, which we don't want to do.

Reply:
Patch https://github.com/GNOME/gtkmm/blob/master/meson.build#L238-L244 to be just `vc14`

Reply:
Tried this, but this seems to assume that the same major.minor versioning scheme was then used in the dependencies as well:

```
tools\extra_defs_gen\meson.build:7:0: ERROR: C++ shared or static library 'glibmm_generate_extra_defs-vc14-2.4' not found
```

Not sure this is really worth starting to make changes across the dependency chain over.

Reply:
Isn't that what we want? We don't want to re-build packages every time we update the visual studio version.